beta/src/pages/learn/referencing-values-with-refs.md

```diff
@@ -1,37 +1,37 @@
 ---
-title: 'Referencing Values with Refs'
+title: 'Refs के ज़रिए वैल्यू का रेफरन्स लेना'
 ---
 
 <Intro>
 
-When you want a component to "remember" some information, but you don't want that information to [trigger new renders](/learn/render-and-commit), you can use a *ref*.
+जब आप किसी कंपोनेंट को कुछ जानकारी "याद" करवाना चाहते हों, लेकिन आप नहीं चाहते कि वह जानकारी [नए रेंडर को ट्रिगर करे](/learn/render-and-commit),तब आप एक *ref * का उपयोग कर सकते हैं।
 
 </Intro>
 
 <YouWillLearn>
 
-- How to add a ref to your component
-- How to update a ref's value
-- How refs are different from state
-- How to use refs safely
+- अपने कंपोनेंट में refs कैसे जोड़ें
```

Comment:
`कंपोनेंट` => `कौम्पोनॅन्ट`

update all the references of this in the whole file. You can do a find and replace